src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -149,6 +149,10 @@ def get_location_type_with_default_from_resource_group(cli_ctx):
     arg_group=sku_arg_group,
     options_list=['--elastic-pool'])
 
+compute_model_param_type = CLIArgumentType(
+    arg_group=sku_arg_group,
+    options_list=['--compute-model'])
```

Comment:
Can you add enum value list here?

Reply:
Actually, I think you don't want to use `get_enum_type` here (see comment at https://github.com/Azure/azure-cli/blob/29527352eafdebf6b39a66f7239cf56507347667/src/azure-cli/azure/cli/command_modules/sql/_params.py#L468 ). CLI core team, is there any way for a parameter to have autocomplete for suggested possible values, but still allow user to enter values that are not suggested?

Reply:
I would say get_enum_type is not bad. If we were to introduce new compute type, we just need to update the enum to take in that from CLI side. it is trivial change to me.

Reply:
I actually think it might be a good idea to add enum type here. Later if we want to introduce new types of offering, we will add it to the enum. Should be trivial change for CLI.